### DISCUSSION TITLE
Can i install with cpenal in shared hosting

### DISCUSSION THREAD

**Discussion by jaainil** [shlinkio/discussions#120] [Help wanted] [answered]
#### How Shlink is set up

* Shlink Version: x.y.z
* PHP Version: x.y.z
* How do you serve Shlink: Self-hosted Apache|Self-hosted nginx|Self-hosted openswoole|Self-hosted RoadRunner|Openswoole Docker image|RoadRunner Docker image
* Database engine used: MySQL|MariaDB|PostgreSQL|MicrosoftSQL|SQLite (x.y.z)

**Comment by acelaya** [comment, marked as answer]
Shared hostings are not the main target for Shlink, so there's no warranty it will work.

Follow the instructions here and see if you are lucky https://shlink.io/documentation/troubleshooting/#how-to-install-shlink-on-a-shared-hosting
